bw2io/strategies/ecospold2.py

```diff
@@ -305,25 +978,95 @@ def has_cpc(exc):
 
 
 def delete_none_synonyms(db):
+    """
+    This function removes None values from the 'synonyms' list of each dataset.
+
+    Parameters
+    ----------
+    db : list
+        A list of datasets, where each dataset is a dictionary containing a
+        'synonyms' key with a list of synonyms.
+
+    Returns
+    -------
+    list
+        The updated list of datasets with None values removed from the
+        'synonyms' list.
+
+    Examples
+    --------
+    >>> db = [
+    ...     {
+    ...         "synonyms": ["synonym1", None, "synonym2"],
+    ...     },
+    ...     {
+    ...         "synonyms": ["synonym3", "synonym4"],
+    ...     }
+    ... ]
+    >>> delete_none_synonyms(db)
+    [
+        {
+            "synonyms": ["synonym1", "synonym2"],
+        },
+        {
+            "synonyms": ["synonym3", "synonym4"],
+        }
+    ]
+    """
     for ds in db:
         ds["synonyms"] = [s for s in ds["synonyms"] if s is not None]
     return db
 
 
 def update_social_flows_in_older_consequential(db, biosphere_db):
-    """The consequential system model automatically generates new biosphere flows with the category ``social`` (even though they aren't social flows) which are not really used and definitely not characterized, and whose UUID seems to change with each release. They are:
```

Comment:
@cmutel, your input on how this could best be describe in the 'Notes' section of the new docstring would be appreciated.

Reply:
I think it is fine as is - we will probably be dropping this soon in any case.

Reply:
ok so I will leave it as deleted